env.yaml

```diff
@@ -11,3 +11,5 @@ dependencies:
   - pip:
     - pre-commit
     - ipywidgets
+    - kaleido
```

Comment:
Where does `kaleido` come into play within this PR (I may have missed this and if so, apologies)?

Reply:
kaleido is what I used as as support package to generate PNG files from the plotly interactive html graphs.